src/TConsole.cpp

```diff
@@ -289,12 +289,12 @@ TConsole::TConsole(Host* pH, ConsoleType type, QWidget* parent)
     splitter->setPalette(splitterPalette);
     splitter->setParent(layer);
 
-    mUpperPane = new TTextEdit(this, splitter, &buffer, mpHost, false);
+    mUpperPane = new TTextEdit(this, splitter, &buffer, mpHost, false, QFontMetrics(mpHost->mDisplayFont));
```

Comment:
:warning: Using `mpHost->mDisplayFont`  is NOT appropriate for all instances of `TTextEdit` you will need to inspect the parent `TConsole`'s (i.e. `mpConsole`) `mType` and work out which font to pass.

Reply:
Ok, I'll try to understand and fix it